package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "jshint *.js **/*.js && jscs *.js **/*.js && nodeunit"
+    "test": "jshint {.,bin,lib,lib/repo,test}/*.js && jscs {.,bin,lib,lib/repo,test}/*.js && nodeunit"
```

Comment:
@markelog any idea how to make this match all js files in the repo, excluding node_modules, without having to spell out each directory?

Also, I tried extract that list into an npm config variable, but the variable replacement won't resolve the wildcards, so it ends up with this:
```sh
ERROR: Can't open {.,bin,lib,lib/repo,test}/*.js
Error: Path {.,bin,lib,lib/repo,test}/*.js was not found.
```